dbt/include/trino/macros/materializations/incremental.sql

```diff
@@ -86,9 +86,9 @@
     {% endif %}
 
     {#-- Get the incremental_strategy, the macro to use for the strategy, and build the sql --#}
-    {% set incremental_predicates = config.get('incremental_predicates', none) %}
+    {% set incremental_predicates = config.get('predicates', none) or config.get('incremental_predicates', none) %}
```

Comment:
Note that this change is rather characteristic for dbt-core 1.4.1 and not dbt-core 1.4.0

See https://github.com/dbt-labs/dbt-core/commit/e8da84fb9e177d9eee3d7722d3d6906bb283183d

Reply:
Actually this PR is about to adjust to 1.4.1